frontend/client/src/screens/Settings.js

```diff
@@ -15,6 +16,17 @@ import ResetPasswordModal from '../components/ResetPasswordModal'
 import CircularProgress from '@material-ui/core/CircularProgress'
 
 const MAXFILESIZE = 10 * 1024 * 1024
+const theme = createMuiTheme({
+  breakpoints: {
+    values: {
```

Comment:
Good work, let's just add a comment explaining that these are set to correspond to the _include-media.scss breakpoints.

Reply:
Sounds good, I'll add it in